.github/workflows/inspektor-gadget.yml

```diff
@@ -289,6 +289,8 @@ jobs:
       - btfgen
       - build-helper-images
     runs-on: ubuntu-latest
+    permissions:
+      security-events: write
```

Comment:
Should we also remove the ones on btfgen? I think there were misplaced in https://github.com/inspektor-gadget/inspektor-gadget/commit/f04d95b078d5eb6eee289cedd27dc38f7aa0d9e6